docs/backend/upgrading/version-specific-migration/upgrade-to-60.md

```diff
@@ -294,4 +294,251 @@ The same change is needed for the no longer supported `includeDependenciesOverri
 ```{seealso}
 -   [PLIP 3339](https://github.com/plone/Products.CMFPlone/issues/3339)
 -   [`plone.autoinclude`](https://github.com/plone/plone.autoinclude) documentation
-```
\ No newline at end of file
+```
+
+
+(v60-dexteritytextindexer-label)=
+
+## `collective.dexteritytextindexer` merged
+
+You can mark fields of a content type as searchable.
+You can then find an object in the search form by using any text from these fields.
+For this to work, you need to enable the `plone.textindexer` behavior on the content type.
+For more information, see {ref}`backend-indexing-textindexer-label`.
+
+In earlier Plone versions, this feature was available in the `collective.dexteritytextindexer` package.
+This was merged into Plone 6.0 core, with most code ending up in `plone.app.dexterity`.
+
+If you have an add-on or custom code that uses the old package, then this must be upgraded.
+These are the needed changes:
+
+- Remove `collective.dexteritytextindexer` from the `install_requires` in `setup.py`.
+- Update the Python code of your content type code, like this:
+
+```diff
+-from collective import dexteritytextindexer
++from plone.app.dexterity import textindexer
+ from plone.app.textfield import RichText
+ from plone.supermodel import model
+...
+ class IExampleType(model.Schema):
+-    dexteritytextindexer.searchable("text")
++    textindexer.searchable("text")
+     text = RichText(...)
+```
+
+- In the GenericSetup xml of your content type, use the new behavior:
+
+```diff
+ <property name="behaviors" purge="false">
+-  <element value="collective.dexteritytextindexer" />
+-  <element value="collective.dexteritytextindexer.behavior.IDexterityTextIndexer" />
++  <element value="plone.textindexer" />
+ </property>
+```
+
+- Search for any left-overs of `collective.dexteritytextindexer` in your code and replace it.
+
+The inline upgrade to Plone will replace the two versions of the old behavior with the new one in all content types.
```

Comment:
Is "in-place" what you mean? See https://deploy-preview-1340--6-dev-docs-plone-org.netlify.app/backend/upgrading/intro.html#in-place-migrations

```suggestion
The in-place upgrade to Plone will replace the two versions of the old behavior with the new one in all content types.
```